### ISSUE TITLE
Dash not initializing on failed autologin

### ISSUE THREAD

**Issue by coolymike** [Yellow-Dog-Man/Resonite-Issues#2239]
### Describe the bug?

When a user launches Resonite, and Resonite fails to log in automatically, the dash does not initialize. This makes the game completely unplayable for certain users.

### To Reproduce

- Log into Resonite
- Close the game, and get logged out. This can be due to the auto logout 30 day timer, a password change, etc.
- Reopen the game on the latest version.
- Dash does not initialize.

### Expected behavior

When autologin fails, the dash should initialize as if a login was never attempted.

### Screenshots

On game startup:
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/11460382/c7317a47-6d05-4ae5-867c-c6c78051cfb6)

### Resonite Version Number

2024.6.5.198

### What Platforms does this occur on?

Windows, Linux

### What headset if any do you use?

Desktop

### Log Files

[Broken_Main_Menu_PC_-_2024.6.5.198_-_2024-06-05_03_18_34.log](https://github.com/user-attachments/files/15580777/Broken_Main_Menu_PC_-_2024.6.5.198_-_2024-06-05_03_18_34.log)

The bit that's shared among logs of other users with this issue is:
```
Error logging in: BadRequest
Login.InvalidCredentials
```

### Additional Context

The update log states:
> Workspaces will not run the default initialization when the user is being auto logged in

This is likely the cause of the issue, as a failed autologin could still prevent the dash from initializing.

### Reporters
Github: 
@energymaster22
@coolymike 
@fennboii

Discord:
```
@energymaster
@whippersnatchpumpkinpatch
@fennboii
```

**Comment by Frooxius** [comment]
This is a bit odd. Why do they need workarounds to launch Resonite?

Does the dash not load on the next launch when the auto login is invalidated?

**Comment by coolymike** [comment]
The dash not not load when an autologin is attempted and fails. The autologin does not "invalidate itself", and will be attempted on the next launch, even if it failed.

This means once a user is logged out from outside the game, Resonite will keep attempting to autologin, fails, and does not load the dash.

**Comment by Frooxius** [comment]
Ah I see. Thanks for the extra context. I'll have a look into this.

**Comment by Frooxius** [comment]
Ok this should be fixed in 2024.6.5.1084, thanks!
